src/assets/js/statusPage.js

```diff
@@ -36,17 +38,22 @@ export default class StatusPage {
       fetch(statusEndpoint)
         .then(res => res.json())
         .then(data => {
-          console.info('StatusPage: polled request and got a status of: ' + data.status)
+          logger.info('StatusPage: polled request and got a status of: ' + data.status, { type: types.App })
```

Comment:
I think your acceptance tests are failing because of the use of the logger in this file. 
reverting this file's changes (back to using console), building and running the tests leads to them passing for me. 

statusPage.js is run in the clients browser so perhaps it doesn't have the ability to use logger??

Reply:
for a little more context, manually performing the actions the test performs eventually leads me to this client side javascript error

![image](https://github.com/user-attachments/assets/006a4f17-43dc-4ab8-a976-0584aced06b5)

Reply:
doh, good catch. Should have try that first, but the other issue was derailed me.